workspace/php2/src/XSolveSecurityBundle/Controller/DefaultController.php

```diff
@@ -1,107 +1,15 @@
-            $error = $session->get(SecurityContext::AUTHENTICATION_ERROR);
-            $session->remove(SecurityContext::AUTHENTICATION_ERROR);
-        }
 
-        return $this->render(
-                        'XSolveSecurityBundle:Security:login.html.twig', array(
-                    // last username entered by the user
-                    'last_username' => $session->get(SecurityContext::LAST_USERNAME),
-                    'error' => $error,
-                        )
-        );
-    }
-    public function adminUsersAction() {
-        
-    $product = $this->getDoctrine()
-        ->getRepository('XSolveSecurityBundle:User')
-        ->findAll();
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-        return $this->render('XSolveSecurityBundle:Default:adminUsers.html.twig', array(
-                    'table' => $product
-                        ));
-    }
-    public function page1Action() {
-        return $this->render('XSolveSecurityBundle:Default:page1.html.twig');
-    }
-    public function page2Action() {
-        return $this->render('XSolveSecurityBundle:Default:page2.html.twig');
-    }
-    public function page3Action() {
-        return $this->render('XSolveSecurityBundle:Default:page3.html.twig');
-    }
-    public function PanelAdminAction() {
-        return $this->render('XSolveSecurityBundle:Default:admin.html.twig');
-    }
-    public function registerAction(Request $request){
-        $username = $form["username"]->getData();
-        $password = $form["password"]->getData();
 
-        // create a task and give it some dummy data for this example
-        $task = new User();
-        $task->setUsername('user');
-        $task->setPassword('user');
-        
-        $form = $this->createFormBuilder($task)
-            ->add('username', 'text')
-            ->add('password', 'text')
-            ->add('save', 'submit')
-            ->getForm();
-        $form->handleRequest($request);
-        
-        if ($form->isValid()) {
-        // perform some action, such as saving the task to the database
-            $em = $this->getDoctrine()->getManager();
-            $em->persist($task);
-            $em->flush();
-        return $this->redirect($this->generateUrl('login'));
-        }
-        
-        
-        return $this->render('XSolveSecurityBundle:Security:register.html.twig', array(
-            'form' => $form->createView(),
-        ));
-    }
-    public function ShowUserAction($id) {
+class DefaultController extends Controller {
 
-      $user = $this->getDoctrine()
-	      ->getRepository('XSolveSecurityBundle:User')
-	      ->find($id);
+   public function indexAction() {
 
-      if (!$user) {
-	 throw $this->createNotFoundException(
-		 'No product user for id ' . $id
-	 );
-      }
-      return $this->render('XSolveSecurityBundle:Default:showUser.html.twig', array(
-		  'user' => $user
-      ));
+      return $this->render('XSolveSecurityBundle:Default:index.html.twig');
```

Comment:
W jakim celu został pozostawiony ten ``Controller`` z tą akcją ``indexAction`` i z tym szablonem twiga ``XSolveSecurityBundle:Default:index.html.twig``.

Ogólnie mamy taką zasadę że nie pozostawiamy ``legacy code`` który jest na obecny moment niepotrzebny, ja nie wiem czy jest to potrzebne, ale jak nie to dobrze by było jakby wszystkie powiązane rzeczy pousuwać.